docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     build:
       context: .
     ports:
-      - 3500:3500
+      - "3500:3500"
```

Comment:
Wat werkte er bij jou niet meer? Gezien dit eerder wel werkte... We hebben in de andere repo's hier ook zelden quotes omheen. Poortnummers lager dan 60 zijn een probleem (https://stackoverflow.com/a/59743942/17219777), maar dat is hier niet het geval 😉

Reply:
Dit is een aanbeveling vanuit mijn IDE.